lib/msf/core/opt_condition.rb

```diff
@@ -16,6 +19,36 @@ def self.eval_condition(left_value, operator, right_value)
       end
     end
 
+    # Format an option's conditions as a human readable string
+    # @param [Msf::Module] mod The module module
+    # @param [Msf::OptBase] opt the option which has conditions present
+    # @return [String]
+    def self.format_conditions(_mod, opt)
+      left_source = opt.conditions[0]
+      operator = opt.conditions[1]
+      right_value = opt.conditions[2]
+      expects_blank_values = Array(right_value).all?(&:blank?)
+
+      case operator.to_sym
+      when :==, :!=
+        "#{left_source} #{operator} #{right_value}"
+      when :in
+        if expects_blank_values
+          return "#{left_source} is blank"
+        end
+
+        "#{left_source} #{operator} #{right_value.join(',')}"
+      when :nin
+        if expects_blank_values
+          return "#{left_source} is not blank"
+        end
+
+        "#{left_source} not in #{right_value.join(',')}"
+      else
+        'placeholder'
+      end
+    end
```

Comment:
We need to apply the feedback from here:

https://github.com/rapid7/metasploit-framework/pull/17505/files#r1082926728

https://github.com/rapid7/metasploit-framework/pull/17505/files#r1082930108

https://github.com/rapid7/metasploit-framework/pull/17505/files#r1082930627

Reply:
Got all of those addressed